Example/package.json

```diff
@@ -13,7 +13,7 @@
   "dependencies": {
     "@react-native-community/masked-view": "^0.1.10",
     "@react-navigation/bottom-tabs": "^5.6.1",
-    "@react-navigation/core": "^5.11.1",
+    "@react-navigation/core": "^5.14.4",
```

Comment:
Should we increase the code version as well?

Reply:
Not required. Didn't see any critical fix, just tried a new version before digging into details, thought it might be fixed with a new version :)